tests/models/compute/server_unit_tests.rb

```diff
@@ -1,6 +1,6 @@
 require_relative '../../helper'
 
-require 'fog/vsphere/models/compute/server'
+# require 'fog/compute/vsphere/models/server'
```

Comment:
Can this line be removed?

Reply:
I believe so, can not confirm though as the tests are broken. Would be maybe good idea to make them pass first and then continue on this one?

Reply:
That sounds like an excellent idea.